CONTRIBUTING.md

```diff
@@ -93,3 +93,9 @@ Please note that the recommended local port for the services are different than
     - All dependencies must be abstracted as an interface. This will make it easier to switch libraries at later point
       of time.
 * Version tagging as per Semantic versioning (https://semver.org/)
+
+### Documentation
+* After adding or modifying code existing code, update the documentation too - [development/docs](https://github.com/gofr-dev/gofr/tree/development/docs).
```

Comment:
It can be more thorough on how it has to be updated.
For example, it can include about adding a completely new documentation where, it also needs to be updated in navigation.js.
2. md formatting rule where, #, ##, ### ... has to be used in proper sequence.